metricbeat/module/elasticsearch/_meta/docs.asciidoc

```diff
@@ -0,0 +1,4 @@
+== elasticsearch Module
+
+This is the elasticsearch Module. The elasticsearch module contains a minimal set of metrics to enable monitoring of elasticsearch across multiple versions.
```

Comment:
Should we link to xpack monitoring here, as a more complete alternative? Can come in a different PR.